### ISSUE TITLE
IOS: View is not in the window hierarchy

### ISSUE THREAD

**Issue by PanRada** [kyunkakata/react-native-tiktok-sign#1]
Hi

Wondering if u fork will going to fix that problem as I notice on version RNN 67.4 auth doesn't work

Is any chance u can look at this and maybe fix ?

**Comment by kyunkakata** [comment]
Yeah, I created this fork in order to fix that problem. I will try the last 5.0.14 TiktokOpenSDK to make sure that Auth will work even when Tiktok app is not installed. 
If it is doesn't work, the alternative way is using react-native-webview and try to create function same as native one.
But right now, I don't have time for this. Will be updated in the next 2 weeks or more.
